mapping/svg-maps.md

```diff
@@ -0,0 +1,15 @@
+# About SVG files
+SVG files, or scalable vector graphic files, are a unique image type that work well for making maps. Maps made with SVG files allow us access to the discrete geographies within the map using JavaScript, which also opens up client-side map rendering. This makes SVG maps a powerful tool when working with front-end frameworks like React.js.
+
+## SVG Maps
+In order to build an SVG map, you first need to find an open-source SVG image of the map you want to use. The first place you can look is the [`svg-maps`](https://github.com/VictorCazanave/svg-maps) repo, which complements the [`react-svg-map`](https://www.npmjs.com/package/react-svg-map) repo. These repos provide functionality for using SVG maps with React. If the map you want to use is not already a part of the `svg-maps` repository, [Wikimedia Commons](https://commons.wikimedia.org/w/index.php?search=filemime%3Aimage%2Fsvg%2Bxml&title=Special%3ASearch&profile=advanced&fulltext=1&advancedSearch-current=%7B%22fields%22%3A%7B%22filetype%22%3A%22image%2Fsvg%2Bxml%22%7D%7D&ns6=1) is a good place to start looking for an SVG map.
```

Comment:
I think it would be more clear to frame this doc around a recommendation to use the `react-svg-maps` library. What do you think about reorganizing what you've already written along these lines?

- `react-svg-maps`
    - Brief description of the library and where to source SVG files
- When to use SVG maps, and when to consider other tools
- Examples
- Further reading
    - More on the SVG file format
    - How to edit SVG files